### PR TITLE
SISRP-33357 - Two broken links in My Campus (DNS and 404)

### PR DESCRIPTION
**Pull request by nkutub**
https://jira.berkeley.edu/browse/SISRP-33357

* change two links in campuslinks.json

### CODE REVIEW

**Comment by redconfetti** [comment]
@nkutub - Is the 3rd link that is being changed a CS Link API link, or is it changed by some process other than by code in our repository?

| ----- | ------------------------------------------ |
| old:   | New Student Services (Including CalSO) |
| new  | New Student Services (including Golden Bear Orientation) |
| link   | http://orientation.berkeley.edu/ |
| ----- | ------------------------------- |

**Comment by nkutub** [comment]
@redconfetti , Third link updated, good catch!

File: public/json/campuslinks.json [hunk 1]
```diff
@@ -2944,6 +2925,7 @@
         }
       ]
     },
+    },
```
Comment (davidlimcheng) [inline public/json/campuslinks.json]: @nkutub I believe this bracket-close needs to be deleted?

Comment (redconfetti) [inline public/json/campuslinks.json]: I second this.

Comment (nkutub) [inline public/json/campuslinks.json]: @davidlimcheng, all fixed

**Pull request merged**